cmd/root.go

```diff
@@ -0,0 +1,154 @@
+package cmd
+
+import (
+	"bufio"
+	"fmt"
+	"github.com/spf13/cobra"
```

Comment:
The import from the standard library and github.com should be separated (like in https://github.com/yuzutech/kroki-go/blob/master/http.go#L3 for example).